bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/common/PckGenerator.java

```diff
@@ -777,4 +797,28 @@ private static int timeToRampValue(int timeMSec) {
         }
         return ret;
     }
+
+    /**
+     * Converts duration in milliseconds to lcntimer value
+     * Source: https://www.symcon.de/forum/threads/38603-LCN-Relais-Kurzzeit-Timer-umrechnen
```

Comment:
I honestly don't know how to handle this from the license perspective. I'll clarify this.

Reply:
I'd say it is ok. The code is not copied from there, but clearly self-written as it is a different programming language - so we don't have any copyright issues here.